locales/cy-GB.js

```diff
@@ -0,0 +1,8 @@
+export const val = {
+	'buttonCancel': 'Canslo',
+	'buttonUpdate': 'Diweddaru',
+	'inputLabelPoints': 'Pwyntiau',
+	'mainBodyDescription': 'Bydd diweddaru manylion y cwestiynau yma yn cael eu cymhwyso i'r cwis yn Brightspace ond ni fyddant yn effeithio ar y cwestiwn gwreiddiol a grëwyd yn yr offeryn Dysgu Allanol.',
```

Comment:
The single quote needs to be escaped
```suggestion
	'mainBodyDescription': 'Bydd diweddaru manylion y cwestiynau yma yn cael eu cymhwyso i\'r cwis yn Brightspace ond ni fyddant yn effeithio ar y cwestiwn gwreiddiol a grëwyd yn yr offeryn Dysgu Allanol.',
```